hack/BUILD

```diff
@@ -47,3 +48,22 @@ test_suite(
         #        "verify-gofmt",
     ],
 )
+
+go_binary(
+    name = "hack",
```

Comment:
this one is probably better named `e2e` but I'm guessing gazel wouldn't support that.